tv_apps/fling-player/js/simple_key_nav_helper.js

```diff
@@ -0,0 +1,70 @@
+/* global SimpleKeyNavigation */
+(function(exports) {
+  'use strict';
+
+  /**
+   * This a simple wrapper around SimpleKeyNavigation for providing some
+   * convinent extra operations on SimpleKeyNavigation
+   *
```

Comment:
It seems you add more abilities to this library, why don't you just add them upon `SimpleKeyNavigation`?